### PR TITLE
Auto fetch mobile number on login

### PR DESCRIPTION
**Pull request by amorenew**
I solved this issue #117 
Now you will see the mobile number on the emulator in the Login screen and real device if the sim card has the mobile number pre-exist on it.

### CODE REVIEW

**Comment by 0crat** [comment]
Job [#123](https://github.com/ammaratef45/zold-flutter-client/issues/123) is now in scope, role is `REV`

**Comment by 0crat** [comment]
This pull request [#123](https://github.com/ammaratef45/zold-flutter-client/issues/123) is assigned to @nermeen78[/z](https://www.0crat.com/u/nermeen78), here is [why](https://www.0crat.com/footprint/CGC59KB9B/b95d3cf6-8ae5-42ad-bea8-3f2dc71b75c2); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @ammaratef45[/z](https://www.0crat.com/u/ammaratef45) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be no monetary reward for this job

File: zold_wallet/pubspec.lock [hunk 1]
```diff
@@ -160,6 +160,13 @@ packages:
       url: "https://pub.dartlang.org"
     source: hosted
     version: "1.1.6"
+  mobile_number:
+    dependency: "direct main"
+    description:
+      name: mobile_number
```
Comment (Nermeen78) [inline zold_wallet/pubspec.lock:166]: @amorenew mobile_number plugin for android devices only. So this feature is still not supported in the ios devices. Could you find a solution for that?

Comment (amorenew) [reply zold_wallet/pubspec.lock:166]: @Nermeen78 fetching mobile number is not allowed by Apple policy.

Comment (Nermeen78) [reply zold_wallet/pubspec.lock:166]: @amorenew I already read about the apple policy but there are a lot of ios apps fetch the mobile number in registration like (IMO). 
@ammaratef45  please take a decision in this issue, should we ignore ios devices for applying this feature?

Comment (amorenew) [reply zold_wallet/pubspec.lock:166]: @Nermeen78 For reference https://stackoverflow.com/questions/193182/programmatically-get-own-phone-number-in-ios
As **davidgyoung** answer they send SMS then they checked the device phone number who sent it.
It is so complicated way for our case.

Comment (Nermeen78) [reply zold_wallet/pubspec.lock:166]: @amorenew yes I know there is no easy way to do this in ios but it is possible

Comment (ammaratef45) [reply zold_wallet/pubspec.lock:166]: @Nermeen78 @amorenew 
Thanks, I will accept it as it is for now.
when a new release of the package is delivered we will check what more features we can use from it.

**Review by ammaratef45 — APPROVED** [review]

**Comment by 0crat** [comment]
@sereshqua[/z](https://www.0crat.com/u/sereshqua) please review this job completed by @nermeen78[/z](https://www.0crat.com/u/nermeen78), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by 0crat** [comment]
The job [#123](https://github.com/ammaratef45/zold-flutter-client/issues/123) is now out of scope

**Comment by 0crat** [comment]
There is an unrecoverable failure on my side. Please, submit it [here](https://github.com/zerocracy/farm/issues):

```
PID: 4@d187b2ba-a3c0-4736-9c6e-4a20e5597990, thread: pool-7-thread-7
com.jcabi.xml.ListWrapper[162] com.jcabi.xml.ListWrapper$NodeNotFoundException: XPath '/equity/shares/text()' not found in '<?xml version="1.0" encoding="UTF-8"?>\uA<equity x..174..acy.com/0.65.2/xsd/pm/cost/equity.xsd"/>\uA': Index (0) is out of bounds (size=0)
```

[0.47.6](https://github.com/zerocracy/farm): CID: [97d50b0e-c937-46c2-ad27-b9dc70b6ad6e](https://www.0crat.com/footprint/CGC59KB9B/97d50b0e-c937-46c2-ad27-b9dc70b6ad6e), Type: "Transfer shares"

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://www.zerocracy.com/policy.html#28): +10 point(s) just awarded to @ammaratef45[/z](https://www.0crat.com/u/ammaratef45)

**Comment by sereshqua** [comment]
@Nermeen78 please make sure that you will try to find at least 3 possible issues during next CR

**Comment by Nermeen78** [comment]
@sereshqua the pull request changes are very small so I couldn't find 3 issues in it. I will try it next CR.

**Comment by sereshqua** [comment]
@0crat quality acceptable

**Comment by 0crat** [comment]
> @0crat quality acceptable  ([here](https://github.com/ammaratef45/zold-flutter-client/issues/123#issuecomment-502980344))

@sereshqua There is an unrecoverable failure on my side. Please, submit it [here](https://github.com/zerocracy/farm/issues):

```
PID: 4@d187b2ba-a3c0-4736-9c6e-4a20e5597990, thread: pool-7-thread-4
com.jcabi.xml.ListWrapper[162] com.jcabi.xml.ListWrapper$NodeNotFoundException: XPath '/equity/shares/text()' not found in '<?xml version="1.0" encoding="UTF-8"?>\uA<equity x..174..acy.com/0.65.2/xsd/pm/cost/equity.xsd"/>\uA': Index (0) is out of bounds (size=0)
```

[0.47.6](https://github.com/zerocracy/farm): CID: [d46a66d0-44f2-40b0-8b2e-812335ae7163](https://www.0crat.com/footprint/CGC59KB9B/d46a66d0-44f2-40b0-8b2e-812335ae7163), Type: "Transfer shares"

**Comment by 0crat** [comment]
> @0crat quality acceptable  ([here](https://github.com/ammaratef45/zold-flutter-client/issues/123#issuecomment-502980344))

@sereshqua There is an unrecoverable failure on my side. Please, submit it [here](https://github.com/zerocracy/farm/issues):

```
PID: 4@d187b2ba-a3c0-4736-9c6e-4a20e5597990, thread: pool-7-thread-7
com.jcabi.xml.ListWrapper[162] com.jcabi.xml.ListWrapper$NodeNotFoundException: XPath '/equity/shares/text()' not found in '<?xml version="1.0" encoding="UTF-8"?>\uA<equity x..174..acy.com/0.65.2/xsd/pm/cost/equity.xsd"/>\uA': Index (0) is out of bounds (size=0)
```

[0.47.6](https://github.com/zerocracy/farm): CID: [d991bfa8-e4a2-45c3-b3d1-63340de00ce0](https://www.0crat.com/footprint/CGC59KB9B/d991bfa8-e4a2-45c3-b3d1-63340de00ce0), Type: "Transfer shares"

**Comment by 0crat** [comment]
Order was finished, quality is "acceptable": +15 point(s) just awarded to @nermeen78[/z](https://www.0crat.com/u/nermeen78)

**Comment by 0crat** [comment]
Quality review completed: +4 point(s) just awarded to @sereshqua[/z](https://www.0crat.com/u/sereshqua)

**Pull request merged**